test/commands/code-analyzer/init.test.ts

```diff
@@ -0,0 +1,78 @@
+import {stubSfCommandUx} from '@salesforce/sf-plugins-core';
+import {TestContext} from '@salesforce/core/lib/testSetup';
+import InitCommand from '../../../src/commands/code-analyzer/init';
+import {InitAction} from '../../../src/lib/actions/InitAction';
+
+describe('`code-analyzer init` tests', () => {
+	const $$ = new TestContext();
+	let spy: jest.SpyInstance;
+	let receivedActionInput: object;
+
+	beforeEach(() => {
+		stubSfCommandUx($$.SANDBOX);
```

Comment:
This stubs out the command's UX instance, to prevent anything from being actually logged to the terminal during tests, which keeps the terminal cleaner. As such, it's in the test files for all of the commands now.